lib/tasks/projects.rake

```diff
@@ -71,25 +75,6 @@ namespace :projects do
     end
   end
 
-  desc "Check to see if projects are still removed/deprecated"
-  task check_removed_status: :environment do
```

Comment:
Since we are no longer filtering for `not_removed` in the `check_status` task, we can remove this task entirely.